terraform/ec2/README.md

```diff
@@ -198,6 +198,7 @@ Outputs:
         "ecs:RunTask",
         "ecs:StartTask",
         "ecs:StopTask",
+        "ecs:PutClusterCapacityProviders",
```

Comment:
Unrelated change to add missing permission to ECSEC2 test

Reply:
Also, do we need to remove the elasticfilesystem permissions above? I know this is just a readme and wouldn't affect the existing roles/policies but just want to make sure if those permissions are still required. I'll approve it if that's still in used by some other tests.

Reply:
oh good catch, i forgot to remove those efs permissions

sure I can sort them